tests/test_target.py

```
@@ -2,63 +2,64 @@
 
-        # Chain 2 mines a fast block, difficulty goes to 2
-        block1 = Block(1, chain2.last_block.hash, [], timestamp=chain2.last_block.timestamp + 1, difficulty=chain2.current_difficulty, state_root=chain2.state.state_root())
+        # Chain 2 mines a fast block, target goes to MAX_TARGET - 11
+        block1 = Block(1, chain2.last_block.hash, [], timestamp=chain2.last_block.timestamp + 1, target=chain2.current_target, state_root=chain2.state.state_root())
         mine_block(block1)
         chain2.add_block(block1)
-        self.assertEqual(chain2.current_difficulty, 2)
+        self.assertEqual(chain2.current_target, MAX_TARGET - 11)
         
         # Reorg chain1 to chain2
         success, orphans = chain1.resolve_conflicts(chain2.chain)
```

Ruff (0.16.0)

[warning] Unpacked variable `orphans` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)